.docker.env

```diff
@@ -0,0 +1,11 @@
+# NOTE TO DEVELOPERS
+# Please do not directly change the values of the variables specified in this file, as this may lead to unexpected behavior of your local development server.
```

Comment:
Not sure whether it means, not changing the value in this file?

Reply:
Also, can we ask developers to create this file and .gitignore it as this is more of dev configuration file? I'm worried developer frequently/unexpectedly pushing changes in this file. Maybe have a Make command to create this file.

Makefile:
```
create_defualt_env_file:
    cat "..." > .docker.env

init: create_defualt_env_file build-servers 

...
```

So the initial step for a new contributor would be to run "make init" and it also provides us the space for configuration of any initial steps like pre-push hooks, wdyt?

Reply:
yes, you are correct!..

Reply:
done, added this within `make setup-devserver` command